torchtitan/parallelisms/utils.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+import torch
+
+from torchtitan.logging import logger
+
+
+def check_strided_sharding_enabled() -> None:
+    # Correct 2D/3D DCP usage requires DTensor's strided sharding in PR
+    # https://github.com/pytorch/pytorch/pull/130760. This function checks if users'
+    # PyTorch nightly-build version is newer than 2024-08-09 to make sure this PR is
```

Comment:
probably add a TODO here to remove this check once 2.5 released

Reply:
that makes sense. We can safely assume users won't use an old pytorch at the time when 2.5 is being released.